docs/SUMMARY.md

```diff
@@ -80,6 +80,7 @@
       - [SAML](development/saml.md)
       - [CAS](development/cas.md)
     - [State Resolution]()
+      - [Room DAG concepts](development/room-dag-concepts.md)
```

Comment:
most of this isn't really part of state resolution, and I don't think I agree with @clokep's recommendation to put it just here. what do you both think?

Reply:
It's in the same ballpark 🤷. I was thinking of renaming the category overall too but didn't come up with something better.

Reply:
I think just stick it one level up the tree.